infrastructure/terraform/per_workspace/modules/etl/sds/main.tf

```diff
@@ -303,6 +320,13 @@ module "trigger_update" {
   }
 }
 
+module "schedule_trigger_update" {
+  source              = "./schedule/"
+  lambda_arn          = module.trigger_update.lambda_function.lambda_function_arn
+  lambda_name         = module.trigger_update.lambda_function.lambda_function_name
+  schedule_expression = var.is_persistent ? "rate(15 minutes)" : "rate(1 day)"
```

Comment:
was thinking that the smallest workable interval would be best, since the update trigger will process all available changelog entries and do nothing in the case that there's nothing to process. On top of that if for whatever reason the state lock is in place I was thinking it would be useful to not wait very long until the next update runs

Reply:
that being said, 1 hour or 15 mins is all the same to me - we're definitely not "doing too much" with running every 15 mins